dtcli/scripts/dt.py

```diff
@@ -389,6 +403,62 @@ def upload(**kwargs):
 
 
 
+@extension.command(
+    help="Downloads all schemas from choosen version"
+)
+@click.option(
+    "--version", prompt=True, help="Schema Version e.g. 1.235"
+)
+@click.option(
+    "--tenant-url", prompt=True, help="Dynatrace environment URL, e.g., https://<tenantid>.live.dynatrace.com"
+)
+@click.option(
+    "--api-token", prompt=True, help="Dynatrace API token. Please note that token needs to have the 'Write extension' scope enabled."
+                                     "If using token from file, type 'file'."
+)
+@click.option(
+    "--token-path",
+    default=DEFAULT_TOKEN_PATH, show_default=True,
+    help="Directory where token file can be found. Please note that token needs to have the 'Write extension' scope enabled.",
+)
+@click.option(
+    "--download-dir",
+    default=DEFAULT_DOWNLOAD_DIR, show_default=True,
+    help="Directory where folder schemas will be created with all downloaded files",
+)
+def schemas(**kwargs):
+    if kwargs['api_token'] != "file":
+        token = kwargs['api_token']
+    elif token_load(kwargs['token_path']):
+        token = token_load(kwargs['token_path'])
+    else:
+        token = os.getenv('TOKEN')
+
+    dt = api.DynatraceAPIClient(kwargs['tenant_url'], token=token)
+    version = dt.download_schemas(kwargs['version'], kwargs['download_dir'])
+    print(f"Downloading schemas for version {version}")
+
+
+@extension.command(
+    help="Delete extension from Dynatrace Cluster e.g., custom:com.dynatrace.extension.extension-name"
+)
+@click.option(
+    "--extension", prompt=True, help="Extension name e.g., custom:com.dynatrace.extension.extension-name"
+)
+@click.option(
+    "--tenant-url", prompt=True, help="Dynatrace environment URL, e.g., https://<tenantid>.live.dynatrace.com"
+)
+@click.option(
+    "--token-path",
+    default=DEFAULT_TOKEN_PATH, show_default=True,
+    help="Directory where token file can be found",
+)
+def delete(**kwargs):
+    token = token_load(kwargs['token_path'])

```

Comment:
Nit: can we please use double quotes, especially since it's a key? I don't think we have a linter rule, but... maybe we should? o.0